src/OrbitGl/App.cpp

```diff
@@ -1849,6 +1853,17 @@ static ErrorMessageOr<std::filesystem::path> FindModuleLocallyImpl(
                         module_data.file_path()));
   }
 
+#if WIN32
+  ErrorMessageOr<std::filesystem::path> symbols_path = orbit_windows_utils::FindDebugSymbols(
```

Comment:
I am a bit surprised about this addition here in this PR. Why was that not needed before?